package.json

```diff
@@ -297,9 +304,8 @@
     "@valist/sdk": "^2.9.1",
     "@vitejs/plugin-react": "^2.2.0",
     "cross-env": "^7.0.3",
-    "electron": "26.2.2",
+    "electron": "^25.8.2",
```

Comment:
we can't do this or we'll break the MetaMask integration. i'll take a look at the errors from 26.2.2 on this branch. It is working on main

Reply:
TRied with another node version, recloned, removed everything, cleaned cache and I still have the issue.
I am suspecting it is some bug with AMD GPUs because I am using a 7900XTX on my PC, I will try it on my laptop.

With the build I get a white screen as well.

Might also be a Wayland bug, so I will try on X11.

But in any case, this will affect a good portion of users on linux and we would need for electron to fix it.